DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -10,4 +10,168 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+// Sets the page to always display the first two elements when loading the page.
+var currentPage = 0;
+
+google.charts.load('current', {'packages':['bar', 'table']});
+
+// Retrieves a list of avaliable keyword campaigns the user has stored and is used in
+// dropdown menu.
+function getKeywordCampaigns() {
+	const keywordCampaignList = document.getElementById('keyword-campaigns');
+    keywordCampaignList.innerHTML = '<option value=0>Select a keyword campaign</option>';
+
+    fetch('/keyword-campaigns').then(response => response.json()).then(keywordCampaigns => {
+        keywordCampaigns.forEach(keywordCampaign => {
+            keywordCampaignList.innerHTML += '<option value=' + keywordCampaign.keywordCampaignId + '>' + keywordCampaign.name  + '</option>';
+        });
+    });
+}
+
+// Checks if user has selected a keyword campaign. If the user has selected a keyword
+// then the for loop draws 2 graphs and creates pagination to explore other charts.
+function getDSACampaigns() {
+    const dsaCampaignsList = document.getElementById('DSA-campaigns');
+    dsaCampaignsList.innerHTML = '<p>First select a keyword campaign.</p>';
+    var keywordCampaignId = document.getElementById("keyword-campaigns").value;
+
+    if (keywordCampaignId != 0) {
+        dsaCampaignsList.innerHTML = '';
+
+        fetch('/DSA-campaigns?keywordCampaignId=' + keywordCampaignId).then(response => response.json()).then(DSACampaigns => {
+
+            var isNumberOfChartsOdd = (DSACampaigns.length % 2);
+
+            // chartsToShow checks what the active page and give a starting
+            // point to display the next two charts.
+            var chartsToShow = currentPage * 2;
+
+            // chartCounter keeps track of which number each chart displayed is
+            // (useful for drawing the graph) and is used as a flag to check if 
+            // two charts have been processed.
+            var chartCounter = 1;
+
+            while (chartCounter <= 2) {
```

Comment:
Make the 2 be a constant named numChartsPerPage, otherwise it seems to be a magic number: https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Thanks for the link!. Done.